src/Workspaces/Core/Portable/Rename/ConflictResolution.cs

```diff
@@ -37,8 +37,10 @@ namespace Microsoft.CodeAnalysis.Rename
         private readonly ImmutableDictionary<DocumentId, ImmutableArray<ComplexifiedSpan>> _documentToComplexifiedSpansMap;
         private readonly ImmutableDictionary<DocumentId, ImmutableArray<RelatedLocation>> _documentToRelatedLocationsMap;
 
+#nullable disable warnings
         public ConflictResolution(string errorMessage) : this()
             => ErrorMessage = errorMessage;
+#nullable enable warnings
```

Comment:
@CyrusNajmabadi for a spot check on this change. This PR fixes a bug that caused us to miss nullable field initialization warnings on a struct with a `: this()` initializer.